deps/Makefile

```diff
@@ -769,6 +769,8 @@ UV_OBJ_TARGET := $(build_libdir)/libuv.la
 UV_CFLAGS :=
 ifeq ($(USEMSVC), 1)
 UV_CFLAGS += -DBUILDING_UV_SHARED
+else
+UV_CFLAGS += -std=gnu99
 endif
```

Comment:
I have a more recent commit on the main branch we use that may make this unnecessary